scripts/gitlab/build-linux.sh

```diff
@@ -11,7 +11,7 @@ echo "CC:               " $CC
 echo "CXX:              " $CXX
 #strip ON
 export RUSTFLAGS=" -C link-arg=-s"
-# Linker for crosscomile
+# Linker for crosscompilation
```

Comment:
I'm not sure what the lines below actually do. Can this be removed? cc @TriplEight

Reply:
you mean lines 15 and 16? 
```
echo "_____ Linker _____"
cat .cargo/config
```
It's just the info about the linker.

Reply:
Right, but I think that with this PR it is not so anymore so probably best to remove it